docs/quickstarts/sso-quickstart.md

```diff
@@ -0,0 +1,118 @@
+
+The add-in project that you've created with the Yeoman generator contains sample code for an SSO-enabled task pane add-in. 
+
+- The **./ENV** file in the root directory of the project defines constants that are used by the add-in project.
+- The **./manifest.xml** file in the root directory of the project defines the settings and capabilities of the add-in.
+- The **./src/helpers/documentHelper.js** file ...
+- The **./src/helpers/fallbackauthdialog.html** file ...
+- The **./src/helpers/fallbackauthdialog.js** file ...
+- The **./src/helpers/fallbackauthhelper.js** file ...
+- The **./src/helpers/ssoauthhelper.js** file ...
+- The **./src/taskpane/taskpane.html** file contains the HTML markup for the task pane.
+- The **./src/taskpane/taskpane.css** file contains the CSS that's applied to content in the task pane.
+- The **./src/taskpane/taskpane.js** file contains the Office JavaScript API code that facilitates interaction between the task pane and the Office host application.
+
+## Try it out
+
+1. Navigate to the root folder of the project.
+
+    ```command&nbsp;line
+    cd "My SSO Office Add-in"
+    ```
+
+2. Run the following command to configure SSO for the add-in.
+
+    ```command&nbsp;line
+    npm run configure-sso
+    ```
+
+3. A web browser window will open and prompt you to enter your Azure credentials. Enter your Azure credentials when prompted. These credentials will be used to register a new application in Azure and configure the settings required by SSO.  
+
+4. After you enter your Azure credentials, close the browser window and return to the command prompt. As the SSO configuration process continues, you'll see status messages being written to the console. As described in the console messages, files within the add-in project that the Yeoman generator created are automatically updated with data that's required by the SSO process.
+
+5. When the SSO configuration process completes, run the following command to start the local web server and sideload your add-in.
+
+    > [!NOTE]
+    > Office Add-ins should use HTTPS, not HTTP, even when you are developing. If you are prompted to install a certificate after you run the following command, accept the prompt to install the certificate that the Yeoman generator provides.
+
+    ```command&nbsp;line
+    npm start
+    ```
+
+6. In the Office client application that you chose when creating the add-in project with the Yeoman generator (i.e., Excel, Word or PowerPoint), choose the **Home** tab, and then choose the **Show Taskpane** button in the ribbon to open the add-in task pane. The following image shows this button in Excel.
+
+    ![Excel add-in button](../images/excel-quickstart-addin-3b.png)
+
+7. At the bottom of the task pane, choose the **Get My User Profile Information** button to initiate the SSO process.
+
+8. If you're not alredy signed in to Office, you'll be prompted to sign in. Sign in with the credentials that you want the add-in to use for SSO.
```

Comment:
Bit of a confusion here with steps 8 and 9: If the user is not logged in and they launch the add-in for the first time, then the SSO system quietly fails and branches to the fallback (non-SSO) system. (Note the file name in the title bar of the screenshot.) After the user consents, then the next time they launch the add-in when not logged in, then the SSO system should work: they should get a login prompt but no consent prompt. So, I think the quickstart after step 10 should have the user close the add-in and restart it. Since they will already be signed in, they shouldn't get any prompt at all. They can just get the data right away. We need the quickstart to get them to this point because  the point of SSO is that they don't have to signin to the add-in if they are signed into Office.